src/pages/editor.tsx

```diff
@@ -1,25 +1,105 @@
 import React from "react";
-import styled from "styled-components";
+import styled, { AnyStyledComponent } from "styled-components";
 import { Navbar } from "../components/navbar";
 import "../components/app/normalize.css";
 
-import { background, textPrimary, grey } from "../constants";
+import { background, grey } from "../constants";
 
-const StyledEditor = styled.div`
+const StyledEditor: AnyStyledComponent = styled.div`
   height: 100vh;
   width: 100vw;
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
   justify-items: stretch;
   background: ${background};
   overflow: auto;
 `;
 
+const MainWrapper: AnyStyledComponent = styled.main`
+  display: flex;
+  align-items: flex-start;
+  flex-direction: row;
+  flex-wrap: none;
+  flex: 1;
+`;
+
+const StyledHeader: AnyStyledComponent = styled.div`
+  height: 72px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-items: stretch;
+  align-items: center;
+  justify-content: center;
+  color: white;
+  border: 1px solid ${grey};
+  border-left: none;
+`;
+
+const StyledTitle: AnyStyledComponent = styled.div`
+  height: 42px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-items: stretch;
+  align-items: center;
+  justify-content: center;
+  color: white;
+  border: 1px solid ${grey};
+  border-left: none;
+  border-right: none;
+`;
+
+const StyledLeftHalf: AnyStyledComponent = styled.div`
+  height: 100%;
+  width: 50%;
+  color: white;
+  border-right: 1px solid ${grey};
+`;
+
+const StyledRightHalf: AnyStyledComponent = styled.div`
+  height: 100%;
+  width: 50%;
+  color: white;
+  border-left: 1px solid ${grey};
```

Comment:
```suggestion
```
Since you already added a `border-right` in `StyledLeftHalf`, you don't actually need this left border for `StyledRightHalf`.